chapters/en/slides/module6/module6_03.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+params:
+  dynamictitle: "module6_03"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+
+# Figure composition
+
+Notes:
+In this module we will see how we layout plots together
+instead of showing only one at a time.
+
+---
+
+## Loading in the data
+
+```{python echo=FALSE}
+import altair as alt
+import pandas as pd
+from vega_datasets import data
+
+state_map = alt.topo_feature(data.us_10m.url, 'states')
+state_pop = pd.read_csv('data/us_population_and_coordinates.csv')
+
+choropleth = (alt.Chart(state_map).mark_geoshape().transform_lookup(
+    lookup='id',
+    from_=alt.LookupData(state_pop, 'id', ['population']))
+.encode(color='population:Q')
+.project(type='albersUsa')).properties(height=200, width=300)
+
+us_map = (alt.Chart(state_map).mark_geoshape(color='lightgray', stroke='white')
+          .project(type='albersUsa'))
+
+points = alt.Chart(state_pop).mark_circle().encode(
+    longitude='longitude',
+    latitude='latitude',
+    size='population')
+
+points_on_map = us_map + points
+```
+
+Notes:
+On this slide we are recreating the choropleth and pointplot map
+from the end of the last slide deck.
+We are saving these as variables so that we can use them to practice
+laying out plots on the following slides.
+
+TODO fix so that this code shows up correctly on the slide
+
+---
+
+## Placing plots next to each other allows us to make more direct comparisons
+
+```{python}
+choropleth | points_on_map
+```
+
+Notes:
+To place plot side by side,
+vi can use the pipe (`|`) operator.
+You have already seen us using this occasionally in previous modules.
+
+We could also have used the `alt.hconcat(choropleth, points_on_map)`,
+but it takes less typing to use the operator.
+
+We can see that the legends are stacked to the right of both plots.
+
+---
+
+## Plots can also be concatenated vertically
+
+```{python}
+choropleth & points_on_map
+```
+
+Notes:
+To place plots next to each other vertically,
+we can use the ampersand (`&`) operator.
+
+We could also have used the `alt.vconcat(choropleth, points_on_map)`,
+but it takes less typing to use the operator.
+
+---
+
+## Vertical and horizontal concatenation can be combined
+
+```{python}
+bars = alt.Chart(state_pop).mark_bar().encode(
```

Comment:
Can we plot something other than population for these bars? Otherwise it is redundant? Or perhaps even better, is keep the bars with pop'n, but plot something like asma cases per 1000 on the clorepleth map?

Also, wouldn't it be better to stack the maps vertically, and then have a vertical bar chart so that you can easily read the state names? You might want to combine some states into other to make this work, but I think that would be most effective.